client/styles/main.css

```diff
@@ -23,3 +23,9 @@
 .w-72 {
     width: 18rem;
 }
+
+.ellipsis {
```

Comment:
Why does this not follow the convention ?

Reply:
I no longer need the ellipsis, I would be removing it